Source/Core/Core/PowerPC/MMU.cpp

```diff
@@ -320,7 +320,8 @@ static void WriteToHardware(u32 em_address, const T data)
       (em_address < (0xE0000000 + Memory::GetL1CacheSize())))
   {
     const T swapped_data = bswap(data);
-    std::memcpy(&Memory::m_pL1Cache[em_address & 0x0FFFFFFF], &swapped_data, sizeof(T));
+    std::memcpy(&Memory::m_pL1Cache[em_address & Memory::GetL1CacheMask()], &swapped_data,
```

Comment:
This changed from `0x0FFFFFFF` to `Memory::GetL1CacheMask()`, but every other occurrence used to be `0x3FFFF`. Is this correct, or should this be `Memory::GetExRamMask()` instead?

Reply:
It certainly shouldn't be `GetExRamMask()`

Taking a better look at this code (and the code ExRam above), it doesn't actually implement mirroring, as the if statement is only matches for the first mirror.

All the `& 0x0fffffff` is doing is lopping off the `0x10000000` or `0xe000000`